src/classes/RD_RecurringDonations.cls

```diff
@@ -522,14 +624,19 @@ public with sharing class RD_RecurringDonations {
                 ERR_Handler.processErrors(errors.errorRecords, ERR_Handler_API.Context.RD.name());
             }            
             
-            if (!system.isBatch() && !system.isFuture() && !RD_ProcessControl.batchButton)
+            if (updateRecurringDonationSet.size() > 1 && !system.isBatch() && !system.isFuture() && !RD_ProcessControl.batchButton)
```

Comment:
Are the changes in lines 616 and 627 what fixes https://github.com/SalesforceFoundation/Cumulus/issues/1114?